tutorials/navigation/navigation_using_navigationmaps.rst

```diff
@@ -33,13 +33,33 @@ The 3D default navigation map RID can be obtained with ``get_world_3d().get_navi
     func _ready() -> void:
         var default_navigation_map_rid: RID = get_world_2d().get_navigation_map()
 
+ .. code-tab:: csharp 2D C#
+
+    public partial class MyClass : Node2D
```

Comment:
We usually follow the pattern `My{BaseClass}` for naming classes in the docs, unless the GDScript version names them explicitly with `class_name`.

```suggestion
    public partial class MyNode2D : Node2D
```